pom.xml

```diff
@@ -128,8 +128,8 @@
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
 					<configuration>
-						<target>1.6</target>
```

Comment:
Why are you updating to Java 8 here?

Reply:
Create a separate PR for this

Reply:
I used Java 8 because it supports lambda expressions. If I had used Java 6, we would need to rework on this when upgrading to Java 8.

Reply:
Create a separate PR to update to Java 8. After it is merged into the master branch, update this PR.